packages/ui/src/badge.tsx

```
@@ -0,0 +1,78 @@
+import { clsx } from "clsx";
+
+import React from "react";
+import { clsx } from "clsx";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'clsx'. Consider to delete it or rename it.

'clsx' is defined here:


(lint/suspicious/noRedeclare)